ui/app/src/Utils/Views.elm

```diff
@@ -47,19 +47,43 @@ checkbox name status msg =
         ]
 
 
-validatedFormField : String -> Result ( String, String ) String -> String -> (String -> msg) -> Html msg
-validatedFormField labelText validatedString classes msg =
-    case validatedString of
-        Ok inputValue ->
+validatedField : (List (Attribute msg) -> List (Html msg) -> Html msg) -> String -> String -> (String -> msg) -> msg -> ValidatedField -> Html msg
+validatedField htmlField labelText classes inputMsg blurMsg field =
+    case field.validationState of
```

Comment:
@stuartnelson3 another class is applied to the input field (`form-control-success` or `form-control-danger`), plus there is an error message in the error case. I think this is fine.